agent/util-scripts/pbench-tool-meister-start

```diff
@@ -340,6 +342,13 @@ def main(argv):
         ), f"bad channel: {resp!r}"
         assert resp["data"] == 1, f"bad data: {resp!r}"
 
+    # 2.5. Add tool metadata json to redis
+    tm_start_path = Path(
+        Path(__file__).resolve(), "..", "..", "tool-scripts", "meta.json"
+    ).resolve(strict=True)
```

Comment:
Can you we add a `try/except` to provide a command line error that is a bit more user-friendly?